src/metabase/db/metadata_queries.clj

```diff
@@ -104,6 +112,24 @@
    (field-query field {:breakout [[:field (u/the-id field) nil]]
                        :limit    (min max-results absolute-max-distinct-values-limit)})))
 
+;; TODO this function and field-distinct-values should be in metabase.models.field-values
```

Comment:
Why did you leave this todo? You just moved this function here so I'm puzzled why there would be a TODO to move it somewhere else. If there's a reason why this is hard, maybe leave it in a comment so the next guy that reads this isn't puzzled either